pkg/reconciler/common/releases.go

```diff
@@ -213,6 +237,20 @@ func FetchManifest(path string) (mf.Manifest, error) {
 	return result, err
 }
 
+// FetchManifestNoCache returns the manifest by reading them from the path, and saves them in the cache.
+func FetchManifestNoCache(path string) (mf.Manifest, error) {
+	result, err := mf.NewManifest(path)
+	if err == nil {
+		cache[path] = result
```

Comment:
So if the cache already had results you will overwrite and not return from cache?

Reply:
Or maybe you are checking that somewhere else?

Reply:
This function always reads from the path, but save it into the cache.
Operator needs to know what has been installed in the cluster. This is why we still save it into the cache.